examples/plugin-for-example/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
something new: can we also publish the generated wat? It may be helpful for users to observe the generated wat, and they can also run it from wasmer. 

`wapm install wabt && wapm run wasm2wat ./target/wasm32-unknown-wasi/release/plugin-for-example.wasm > ./examples/plugin-for-example.wat` should do the trick! 🎩 🌠

Reply:
As discussed, I'm not going to include it because it's too long and hard to read (It's almost entirely WASI code);

I'll add a line to the example readme about it though